ThirdPartyAdapters/vungle/vungle/src/main/java/com/google/ads/mediation/vungle/VungleMediationAdapter.java

```diff
@@ -150,6 +149,8 @@ public void loadRewardedAd(
           mediationAdLoadCallback) {
     mMediationAdLoadCallback = mediationAdLoadCallback;
 
+    Context context = mediationRewardedAdConfiguration.getContext();
```

Comment:
Was this not required

Reply:
@palrahul I'm not sure how it passed their QA but I believe admob accidentally removed this line: 
https://github.com/googleads/googleads-mobile-android-mediation/commit/28738e9f2b9fc446f1f95587a2135e7504c5b1a0#diff-c1f0849a50e777b58a3b95dffff4041cR152

Reply:
@jiancchen thanks, thats scary and should have broken the builds for AdMob to know about it